VideoWeb/VideoWeb/ClientApp/src/app/shared/models/hearing.vh.ts

```diff
@@ -0,0 +1,86 @@
+import { VHParticipant, VHConference, VHEndpoint } from 'src/app/waiting-space/store/models/vh-conference';
+import { HearingBase } from './hearing-base';
+import { ConferenceStatus, Role } from 'src/app/services/clients/api-client';
+
+/** A replacement for Hearing based on the VH Redux Models */
+export class VHHearing extends HearingBase {
```

Comment:
VHHearing === VideoHearingHearing? lol - don't love the naming convention

Reply:
Me neither, but I was following the VH prefix convention for the redux models. I would love to remove the old Hearing model in the future and rename `VHHearing` to `Hearing`